The released version throws the error below when importing matplotlib:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/matplotlib/__init__.py", line 895, in <module>
    defaultParams = rcsetup.defaultParams = {
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/matplotlib/__init__.py", line 898, in <dictcomp>
    rcParamsDefault[key]),
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/matplotlib/__init__.py", line 674, in __getitem__
    return dict.__getitem__(self, key)
KeyError: 'webagg.port'

For upstream's merged fix see: https://github.com/matplotlib/matplotlib/pull/21662
--- setup.py
+++ setup.py
@@ -205,7 +205,7 @@ def update_matplotlibrc(path):
         idx for idx, line in enumerate(template_lines)
         if "#backend:" in line]
     template_lines[backend_line_idx] = (
-        "#backend: {}".format(setupext.options["backend"])
+        "#backend: {}\n".format(setupext.options["backend"])
         if setupext.options["backend"]
         else "##backend: Agg")
     path.write_text("".join(template_lines))
